### PR TITLE
Fix editorial inconsistencies or inaccuracies starting-a-project.md

### PR DESCRIPTION
**Pull request by Raphael-MOUQUIN**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

-----
Corrections d'ordre grammatical , ainsi que les tournures de phrases employées .

### CODE REVIEW

**Comment by moomoo56** [comment]
في أربعاء، ٧ يوليو، ٢٠٢١ في ٤:٥٨ م، كتب Raphael-MOUQUIN <
***@***.***>:

**Review by Raphael-MOUQUIN — COMMENTED** [review]
Reviewed

**Comment by Raphael-MOUQUIN** [comment]
Approved

**Comment by itsbagpack** [comment]
Hi @Raphael-MOUQUIN, it looks like there is a [failing test](https://github.com/github/opensource.guide/pull/2462/checks?check_run_id=3127416199) associated with your PR. Please fix it and let us know when you have so we can re-review this PR after. Thank you!

**Review by MaryJaneLove4 — APPROVED** [review]
Je to ok

**Comment by MaryJaneLove4** [comment]
Ok

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

File: _articles/fr/legal.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Pour le meilleur ou pour le pire, envisagez de les informer même s'il s'agit d'
...
 
-* **Marques :** Vérifiez que le nom de votre projet [n'est pas en conflit avec les marques existantes](../starting-a-project/#eviter-les-conflits-de-noms). Si vous utilisez les marques de votre propre entreprise dans le projet, vérifiez qu'il ne provoque aucun conflit. [FOSSmarks](http://fossmarks.org/) est un guide pratique pour comprendre les marques dans le contexte de projets libres et open source.
+* **Marques :** Vérifiez que le nom de votre projet [n'est pas en conflit avec les marques existantes](../starting-a-project/#%E9viter-les-conflits-de-noms). Si vous utilisez les marques de votre propre entreprise dans le projet, vérifiez qu'il ne provoque aucun conflit. [FOSSmarks](http://fossmarks.org/) est un guide pratique pour comprendre les marques dans le contexte de projets libres et open source.
```
Comment (kenyonj) [inline _articles/fr/legal.md:128]:
```suggestion
* **Marques :** Vérifiez que le nom de votre projet [n'est pas en conflit avec les marques existantes](../starting-a-project/#éviter-les-conflits-de-noms). Si vous utilisez les marques de votre propre entreprise dans le projet, vérifiez qu'il ne provoque aucun conflit. [FOSSmarks](http://fossmarks.org/) est un guide pratique pour comprendre les marques dans le contexte de projets libres et open source.
```

**Review by Raphael-MOUQUIN — COMMENTED** [review]
ok better to remove accents in URL

**Comment by Raphael-MOUQUIN** [comment]
ok, thanks all for your help & contributions

Have great days

File: _articles/fr/legal.md (hunk 1 above)
Comment (Raphael-MOUQUIN) [reply _articles/fr/legal.md]: just seeing that one right now:

U r right, as mentionned b4 safer/better to remove accents at all in URLs

**Pull request merged**